tests/test_models.py

```diff
@@ -611,6 +611,12 @@ class TestCrossE(cases.ModelTestCase):
     num_constant_init = 1
 
 
+class TestBoxE(cases.ModelTestCase):
```

Comment:
@ralphabb you can run this test with `pytest tests/test_models.py::TestBoxE`. Might need a few updates, but this is the general concept we have to make sure that models "work"

Reply:
I see. I will put this placeholder in for now, and will fill it as I go

Reply:
As of now this test is passing, so that's a good sign!